playbook.tex

```diff
@@ -8,6 +8,14 @@
 \usepackage{lipsum}
 \usepackage[sfdefault]{roboto}
 \usepackage[T1]{fontenc}
+\usepackage{soul}												%to underline links
+\usepackage[hidelinks, urlcolor=blue, colorlinks=true]{hyperref}				%to create nice looking links (blue)
+\hypersetup{
+bookmarks=true,												%to create bookmarks in the PDF
+bookmarksnumbered=true,
+pdftitle={Lorem ipsum Play},
+pdfauthor={ChaosTRUB},
```

Comment:
What does that pdf title do? And whatever it does, it should not have that value! Either use `Lorem ipsum` or something else, but not a combination of both. And also, ChaosTRUB should not be the PDF author here. Maybe, in another PR, we could add commands for the PDF author and title, but until then, this should have some sort of default value.